src/main/scala/cromwell/util/ConfigUtil.scala

```diff
@@ -26,14 +26,20 @@ object ConfigUtil {
 
     def keys = config.entrySet().toSet map { v: java.util.Map.Entry[String, ConfigValue] => v.getKey }
 
-    def getStringOption(key: String): Option[String] = {
```

Comment:
Yes, this whole file should ben lenthallized, which it is already I think, there are just a few new things I added in between.